ckb-bin/src/subcommand/run.rs

```diff
@@ -40,6 +40,8 @@ pub fn run(args: RunArgs, version: Version) -> Result<(), ExitCode> {
             ExitCode::Failure
         })?;
 
+    let _freezer = shared.spawn_freeze();
```

Comment:
I think it starts a bit too early. At least it can be moved after `verify_genesis`.